agent/status.go

```diff
@@ -105,6 +107,42 @@ func (a *Agent) statusHandler() http.HandlerFunc {
 	}
 }
 
+type TriggerEndTaskResp struct {
+	Status  string `json:"status"`
+	Message string `json:"message"`
+}
+
+func (a *Agent) endTaskHandler(w http.ResponseWriter, r *http.Request) {
+	defer func() {
+		_ = grip.GetSender().Close()
+	}()
+
+	grip.Info(message.Fields{
```

Comment:
This message ends up in a local file on the agent, I think, so it probably isn't useful to log errors like this. Instead write the error message in the response to the HTTP call so that the task can log what it gets from the call to /task_status.